src/ate/test_programs/otlib_wrapper/src/lib.rs

```diff
@@ -297,12 +295,8 @@ pub extern "C" fn OtLibTxCpProvisioningData(
 
     // Unpack input strings.
     // SAFETY: The expected input strings must be set by the caller and be valid.
-    let was_cstr = unsafe { CStr::from_ptr(c_was) };
-    let was = was_cstr.to_str().unwrap();
-    let test_unlock_token_str = unsafe { CStr::from_ptr(c_test_unlock_token) };
-    let test_unlock_token = test_unlock_token_str.to_str().unwrap();
-    let test_exit_token_str = unsafe { CStr::from_ptr(c_test_exit_token) };
-    let test_exit_token = test_exit_token_str.to_str().unwrap();
+    let msg_cstr = unsafe { CStr::from_ptr(c_msg) };
+    let msg = msg_cstr.to_str().unwrap();
```

Comment:
Recommend using `.expect("some message")` here so that if the `unwrap` operation ever fails you'll get a nicer panic message.

Reply:
Fixed other occurrences within this function.